### PR TITLE
[docs-beta] remove "what you'll learn" from getting started

### PR DESCRIPTION
**Pull request by cmpadden**
## Changelog

NOCHANGELOG

### CODE REVIEW

**Comment by cmpadden** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24416?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#24416** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24416?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#24415** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24415?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24414** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24414?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24413** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24413?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @cmpadden and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by erinkcochran87** [comment]
I think maybe leave this one - I kinda like it for an intro here.

File: docs/docs-beta/docs/getting-started/quickstart.md [hunk 1]
```diff
@@ -13,14 +13,6 @@ Welcome to Dagster! In this guide, you'll use Dagster to create a basic pipeline
 - Transforms the data
 - Loads the transformed data to a new CSV file
 
-## What you'll learn
```
Comment (PedramNavid) [inline docs/docs-beta/docs/getting-started/quickstart.md:16]: I prefer to keep this one

**Review by PedramNavid — COMMENTED** [review]
I think we don't remove it from here

**Pull request closed without merging**